### ISSUE TITLE
[Enhancement] - Prevent Duplicate "PartialIds" in $ids

### ISSUE THREAD

**Issue by JSv4** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#407]
@pjohnmeyer just put together a nice PR to let us easily switch our $ids and still pass our tests. In part, it looks to see if the partial $id (the relative schema path in the repo, basically) at the end of the $id can be found. 

In our `validator.mjs`, AJV ensures that you can't have duplicate IDs in files with different paths. That said, it's possible to (erroneously) have two files with the same partialIds at the end (this could happen if someone doesn't know what they're doing or perhaps we have old and new schemas being mixed after the $ids get changed for a a release). Do we need an explicit check to ensure this is not the case?

_Originally posted by @JSv4 in https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/pull/406#discussion_r1168741903_

**Comment by pjohnmeyer** [comment]
@JSv4 I don't think this is necessary, probably a problem with my naming just saying "partial". See https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/pull/406/files/e07af324205a0df5a56a89e7fb92659fc7be58f9#r1170089041
